tests/framework/installer/ceph_installer.go

```diff
@@ -104,10 +104,6 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		}
 	}
 
-	if err = h.WipeDisks(namespace, 2); err != nil {
```

Comment:
Is the `yes-really-destroy-data` flag already set somewhere in the tests?

Reply:
It's actually a mistake, I didn't intend to remove this. Unfortunately, it looks like the CI disk gets a filesystem (couldn't find where) but fast wipe won't remove it and I don't think we want to use `complete` since it takes more time. We don't want to add additional runtime in the CI right? We could always give it a try though...

Reply:
The CI is using a raw disk (or sometimes a raw disk defined by a PV), but not sure what filesystem you mean since OSDs can't be configured if there is a filesystem. 

I would expect a `quick` format should be fine for reinstall of the next test suite. And with this PR I thought we could remove this test code and just use the new wipe code.

Reply:
Somehow, the disk has some fs signature :-(, we can do a Complete deletion to remove everything.